src/libData/AccountData/AccountStoreSC.tpp

```diff
@@ -270,7 +270,7 @@ bool AccountStoreSC<MAP>::UpdateAccounts(const uint64_t& blockNum,
       if (!ret_checker) {
         receipt.AddError(CHECKER_FAILED);
       }
-      receipt.SetCumGas(CONTRACT_CREATE_GAS);
+      receipt.SetCumGas(transaction.GetGasLimit() - gasRemained);
```

Comment:
safe math if neccessary

Reply:
gasRemained is initialized with `transaction.GetGasLimit()`, and the deduction afterward are all wrapped with safemath, unless we change the c++ code, it won't underflow